modules/custom/openy_campaign/src/Entity/MemberCheckin.php

```diff
@@ -166,8 +166,8 @@ public function postSave(EntityStorageInterface $storage, $update = TRUE) {
   private function getActiveCampaignMembers() {
     // Get the list of all active campaigns.
     $currentDate = new DrupalDateTime('now');
-    $currentDate->setTimezone(new \DateTimezone(DATETIME_STORAGE_TIMEZONE));
-    $formatted = $currentDate->format(DATETIME_DATETIME_STORAGE_FORMAT);
+    $currentDate->setTimezone(new \DateTimezone(\Drupal\datetime\Plugin\Field\FieldType\DateTimeItemInterface::STORAGE_TIMEZONE));
```

Comment:
Namespaced classes/interfaces/traits should be referenced with use statements

Reply:
fixed https://github.com/ymcatwincities/openy/pull/2174/commits/839562bfa412cf71c9862ca9d9ce54b72f92ff06